assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -17,4 +17,17 @@ export const tests: Tests = {
     independent: true,
     seed: 0,
   },
+  ContributeLandingOneOffOnlyTest: {
+    variants: ['control', 'oneOffOnly'],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    customSegmentCondition: () => window.matchMedia('(max-width: 660px)').matches && (window.location.pathname.indexOf('contribute') > 0), // matches the 'phablet' breakpoint defined in breakpoints.scss
```

Comment:
Why do you need to check the `pathname` here? Is it to avoid putting people in the test who arrive on the support landing page?

Reply:
exactly